isotovideo

```diff
@@ -90,16 +90,18 @@ use constant {
 # Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
 # Details: https://progress.opensuse.org/issues/90371
 use JSON::PP;
+use FindBin '$RealBin';
+use Cwd qw(realpath);
 
 my $installprefix;    # $bmwqemu::topdir
 
 BEGIN {
     # the following line is modified during make install
     $installprefix = undef;
 
-    my ($wd) = $0 =~ m-(.*)/-;
-    $wd ||= '.';
-    $installprefix ||= $wd;
+    my $topdir = realpath "$RealBin";
```

Comment:
I would avoid the additional `realpath`-syscall. If we need to resolve this to the parent directory in the future we can just use `path($RealBin)->dirname` (from `Mojo::File`).

Reply:
We don't need it at all, just adding `/..` can be enough, it would just avoid the dots in `@INC` for readability.

Reply:
Using `path($RealBin)->dirname` will get us nice readability without extra syscall.

Reply:
I removed `realpath` now.